src/Cake.WindowsAppStore/Cake.WindowsAppStore.csproj

```diff
@@ -24,7 +24,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.26.1">
+    <PackageReference Include="Cake.Core" Version="0.30.0">
```

Comment:
I would suggest to build against the lowest version of Cake before the last breaking change, in this case 0.28.0 for maximum compatibility, if the addin doesn't require something from a newer version.